packages/fx-core/src/plugins/solution/fx-solution/question.ts

```diff
@@ -40,9 +40,10 @@ export const NotificationOptionItem: OptionItem = {
 };
 
 export const CommandAndResponseOptionItem: OptionItem = {
-  id: "CommandBot",
+  // For default option, id and cliName must be the same
+  id: "command-bot",
   label: getLocalizedString("core.CommandAndResponseOption.label"),
-  cliName: "command-and-response",
+  cliName: "command-bot",
```

Comment:
The capability name used in adding capability is `command-and-response`, better to keep consistency?

Reply:
Curious why here is still `command-and-response`, https://github.com/OfficeDev/TeamsFx/blob/dev/packages/cli/src/cmds/capability.ts#L302